crates/re_entity_db/src/entity_properties.rs

```diff
@@ -393,10 +393,10 @@ pub enum TimeSeriesAggregator {
     /// Keep both the minimum and maximum values in the range.
     ///
     /// This will yield two aggregated points instead of one, effectively creating a vertical line.
-    #[default]
     MinMax,
 
     /// Find both the minimum and maximum values in the range, then use the average of those.
+    #[default]
     MinMaxAverage,
```

Comment:
I think this can be quite misleading. If you have a plot that oscillates rapidly between 0-1 and you zoom out, then `MinMax` will produce a thick plot that fills the 0-1 range (which is very informative), but `MinMaxAverage` will show a thin plot that sits in `0.5` (very misleading!)

Reply:
Hmmm good point. It's visually more pleasing for sin/cos thing, but we really should solve this with a thicker line instead then

Reply:
killed the commit that did this.

We should seprately consider maybe to just reduce the amount of aggregation?

Reply:
Yeah, using half a pixel as the aggregation size would be a decent workaround if I don't manage to squeeze in line thickness to this egui release

Reply:
I'll try it